### PR TITLE
[ADD] l10n_it_ddt and  l10n_it_ddt_comment_template

### PR DESCRIPTION
**Pull request by eLBati**
This PR adds the following modules:

* **l10n_it_ddt**
* **l10n_it_ddt_comment_template**

and it fixes https://github.com/OCA/l10n-italy/issues/59

### CODE REVIEW

File: l10n_it_ddt/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    _name = 'stock.ddt'
+    _description = 'DdT'
+
+    @api.multi
+    def get_sequence(self):
+        # XXX: allow config of default seq per company
+        return self.env['ir.sequence'].search(
+            [('code', '=', 'stock.ddt')])[0].id
+
+    def _compute_picking_ids(self):
```
Comment (eLBati) [inline l10n_it_ddt/models/stock.py]: @hurrinico this method should be removed

File: l10n_it_ddt/wizard/add_picking_to_ddt.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    ddt_id = fields.Many2one('stock.ddt')
+
+    @api.multi
+    def add_to_ddt(self):
+        pickings = self.env['stock.picking'].browse(
+            self.env.context['active_ids'])
+        for picking in pickings:
+            if picking.ddt_id:
+                raise Warning(
+                    _("Pickings already in ddt"))
```
Comment (eLBati) [inline l10n_it_ddt/wizard/add_picking_to_ddt.py]: Pickings -> Picking
Could you also add reference to picking name? ```Picking %s ...```

The same for the other raise in this method

**Comment by tafaRU** [comment]
@hurrinico, about your last commit: I noticed your remove OCA from author, please re-add it.
Would be nice also use https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Thank you in advance.

**Comment by eLBati** [comment]
```.travis.yml``` is out of date. Please rebase

**Comment by tafaRU** [comment]
> @hurrinico, about your last commit: I noticed you remove OCA from author, please re-add it.

I mean in the manifest file aka ```__openerp__.py```

Thank you in advance.

**Comment by hurrinico** [comment]
@tafaRU i've done what you told me

**Comment by tafaRU** [comment]
thanks @hurrinico

**Comment by eLBati** [comment]
@tafaRU done.
This is :+1: for me

**Comment by tafaRU** [comment]
You can consider adding standard OCA's icon for module: https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.png

**Comment by hurrinico** [comment]
@tafaRU done

**Comment by hurrinico** [comment]
@eLBati @tafaRU done

**Comment by gurneyalex** [comment]
@eLBati currently checking why the cla bot does not see your ICLA. Sorry for the inconvenience.

**Comment by hurrinico** [comment]
Sorry @gurneyalex how can i "sign" the commits from 8e2b425d65d4ac4994079c3d06740c0d8ca48fb7 to b8d17cfd913ad8cbf91c5bd18d9616cf8dcfec6f

**Comment by gurneyalex** [comment]
@hurrinico you need to change your local github config (s/agliebg/agilebg/), and then either rewrite history or add a .mailcap file.

**Comment by gurneyalex** [comment]
I updated the OCA database with @dcorio github login information.

**Comment by gurneyalex** [comment]
same thing for @simahawk (and linked to the company which has an ECLA)

Nothing available for @LeartS and @OpenCode

**Comment by lepistone** [comment]
(fix of author in the history in progress, please don't merge yet)

**Comment by lepistone** [comment]
@hurrinico @eLBati as suggested yesterday on IRC:
1. make sure your local 8.0 branch is up to date (`git checkout 8.0 ; git pull`)
2. checkout your branch
3. `git rebase 8.0 -X theirs`
4. `git rebase -i 8.0`
5. find the commits you want to edit, and change `pick` to `edit` on them
6. save and exit
7. `git commit --amend --author="Author Name <email@address.com>"`
8. `git rebase --continue`
9. `GOTO 7 UNTIL DONE`

**Comment by eLBati** [comment]
@lepistone done

**Comment by tafaRU** [comment]
@hurrinico please see https://github.com/hurrinico/l10n-italy/pull/10 where I applied some improvements.
After merging my PR, please rebase on 8.0, thanks.

**Comment by tafaRU** [comment]
@hurrinico please see https://github.com/hurrinico/l10n-italy/pull/11

**Comment by tafaRU** [comment]
@hurrinico here https://github.com/hurrinico/l10n-italy/pull/14 I added the module ```l10n_italy_ddt_comment_template```.

**Comment by dcorio** [comment]
@eLBati @tafaRU i've just discovered that we have an interesting module here:
https://github.com/OCA/stock-logistics-transport/tree/8.0/stock_shipment_management

should we have a look at it? the "waybill" is very close to our DdT (and to CMR too)

**Comment by eLBati** [comment]
Closing in favour of https://github.com/OCA/l10n-italy/pull/138

**Pull request closed without merging**